manage/import-export.js

```diff
@@ -51,6 +55,40 @@ onDOMready().then(() => {
   });
 });
 
+function showTimeToBackupTooltip() {
+  const wrap = $('#backup');
+  if (wrap) {
+    $('#backup').open = true;
+    wrap.classList.add('show-reminder');
+  }
+}
+
+function setLastBackup({ update = true }) {
+  const wrap = $('#backup');
+  if (update) {
+    localStorage['stylus-last-backup'] = Date.now();
+  }
+  if (wrap) {
+    wrap.classList.remove('show-reminder');
+  }
+  if (!prefs.get('manage.backup.expanded')) {
+    $('#backup').removeAttribute('open');
+  }
+}
+
+function isItTimeToBackup() {
+  // show message every 14 days
+  const timeSinceLastBackup = 14 * (24 * 60 * 60 * 1000);
+  const lastBackup = localStorage['stylus-last-backup'];
```

Comment:
Instead of using `localStorage`, we should use `chrome.storage` API. If we assume that the database is not stable, we shouldn't rely on `localStorage` either.